testsuite/tests/apicast/test_limit_exceeded_metric.py

```diff
@@ -29,14 +30,23 @@ def application(application):
     return application
 
 
-def test_limit_exceeded(api_client):
+@pytest.fixture(scope="session")
+def error_status_code_429(staging_gateway):
```

Comment:
`limit_reached_status_code` or something similar would be probably better

Reply:
Renamed, is it better now?